RELEASES.md

```diff
@@ -1,3 +1,119 @@
+Version 1.73.0 (2023-10-05)
+==========================
+
+<a id="1.73.0-Language"></a>
+
+Language
+--------
+
+- [Add `internal_features` lint for internal unstable features.](https://github.com/rust-lang/rust/pull/108955/)
+- [Uplift `clippy::fn_null_check` lint as `useless_ptr_null_checks`.](https://github.com/rust-lang/rust/pull/111717/)
+- [Make `noop_method_call` warn by default.](https://github.com/rust-lang/rust/pull/111916/)
+- [Support interpolated block for `try` and `async` in macros.](https://github.com/rust-lang/rust/pull/112953/)
+- [Mention style for new syntax in tracking issue template.](https://github.com/rust-lang/rust/pull/113586/)
+- [Make `unconditional_recursion` warning detect recursive drops.](https://github.com/rust-lang/rust/pull/113902/)
+- [Warn on inductive cycle in coherence leading to impls being considered not overlapping.](https://github.com/rust-lang/rust/pull/114023/)
+
+<a id="1.73.0-Compiler"></a>
+
+Compiler
+--------
+
+- [Write version information in a `.comment` section like GCC/Clang.](https://github.com/rust-lang/rust/pull/97550/)
```

Comment:
We should mention something about how to disable this, for people who don't want it.

Reply:
This was added unconditionally, unless perhaps there's an argument to LLVM or the linker to filter it out. And of course the section can be stripped afterward. What do you suggest?

Reply:
If the only way to remove it is to strip it out, then we should tell people that.